### PR TITLE
Extend teads cookieless test

### PR DESCRIPTION
**Pull request by Jakeii**
## What does this change?
Extend Teads cookieless test
## Does this change need to be reproduced in dotcom-rendering ?

- [ ] No
- [x] Yes (please indicate your plans for DCR Implementation)

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [ ] Other (please specify)

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [ ] No
- [ ] Yes (please give details)

### Does this change break ad-free?

- [ ] No
- [ ] It did, but tests caught it and I fixed it
- [ ] It did, but there was no test coverage so I added that then fixed it

### Does this change update the version of CAPI we're using?

- [ ] No, all the existing database files are just fine
- [ ] Yes, and I have [re-run all the tests locally and checked in all the updated data/database/xyz files](https://github.com/guardian/frontend/blob/main/docs/03-dev-howtos/15-updating-test-database.md)

### Accessibility test checklist

- [ ] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
- [ ] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
- [ ] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)

### Tested

- [ ] Locally
- [ ] On CODE (optional)

### CODE REVIEW

**Review by arelra — APPROVED** [review]

**Comment by rtyley** [comment]
Although we don't currently have full access to TeamCity, we can see in GitHub the build on the `main` branch is currently failing (I noticed this after merging https://github.com/guardian/frontend/pull/25803 !):

image

Running Frontend tests locally shows that we get a test failure because `ab-teads-cookieless` has expired (ie because this PR is not yet merged!):

https://github.com/guardian/frontend/blob/fa37be63651c5e43564b4f70dd878016e4367457/common/app/conf/switches/ABTestSwitches.scala#L80-L84

Locally, I've tried running the tests against the main branch again, but with this `jlk/extend-cookieless-test` branch merged in - all tests then pass locally!

Given that, and seeing as this PR has already been approved, I'm going to merge this PR unless there are any objections?

**Comment by Jakeii** [comment]
So sorry @rtyley this got forgotten before the break!

**Pull request merged**
